### DISCUSSION TITLE
Shared Signals Framework support

### DISCUSSION THREAD

**Discussion by SteveBorger** [keycloak/keycloak#28427] [Q&A] [answered]
Hey all,

Just wondering if Keycloak is looking to include Shared Signals Framework anytime in the future? Apple School/Business Manager now supports third party IDPs and would be nice to be able to integrate Keycloak into this.

Thanks

**Comment by codespearhead** [comment, marked as answer]
I suggest closing this discussion and upvoting #14217 .
